helios-services/src/main/java/com/spotify/helios/rollingupdate/DeploymentGroupEventFactory.java

```diff
@@ -37,51 +36,68 @@
     HOSTS_CHANGED
   }
 
-  public Map<String, Object> rollingUpdateTaskSucceeded(final DeploymentGroup deploymentGroup,
-                                                        final RolloutTask task) {
+  private Map<String, Object> createEvent(final String eventType,
```

Comment:
Not sure exactly how it'd look (not sure a class hierachy makes sense, especially since things like failedTask make no sense what-so-ever for rollout-task events), but passing in a metadata `Map<String, Object>` is surely kludgy.